package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-design-tokens",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Are we certain we want to go to 1.0.0 at this point when other repos are not yet released either? There is a breaking change, but to avoid indicating that this package is now good to go public, I'd stay with 0.2.0 for example and jump to 1.0.0 once we have validated the tokens by using them with ui-components.

Reply:
That would be fine for me.

Reply:
Closing this and creating proper PR for that